benchmarks/benchmark_serving.py

```diff
@@ -210,7 +211,7 @@ def sample_hf_requests(
                            streaming=True)
     assert "conversations" in dataset.features, (
         "HF Dataset must have 'conversations' column.")
-    filtered_dataset = dataset.shuffle().filter(
+    filtered_dataset = dataset.shuffle(seed=random_seed).filter(
         lambda x: len(x["conversations"]) >= 2)
```

Comment:
```suggestion
    filter_func = lambda x: len(x["conversations"]) >= 2
    filtered_dataset = dataset.shuffle(seed=random_seed).filter(filter_func)
```
@wukaixingxp This modification should fix the lint errors.